be/src/agent/task_worker_pool.cpp

```diff
@@ -360,8 +360,9 @@ void* TaskWorkerPool::_create_tablet_worker_thread_callback(void* arg_this) {
         } else {
             ++_s_report_version;
             // get path hash of the created tablet
-            auto tablet = StorageEngine::instance()->tablet_manager()->get_tablet(create_tablet_req.tablet_id);
-            DCHECK(tablet != nullptr);
+            auto st = StorageEngine::instance()->tablet_manager()->get_tablet(create_tablet_req.tablet_id);
+            DCHECK(st.ok());
```

Comment:
It's possible to have an error status here, cannot assume the status is always ok.